cmd/main.go

```diff
@@ -302,18 +306,35 @@ func main() {
 		crdUtils = controllers.NewDummyCRDUtils()
 	}
 
+	gc := garbagecollector.NewGarbageCollectors(db, log.WithField("pkg", "garbage_collector"), hostApi, clusterApi, objectHandler, lead)
```

Comment:
can we have enable / disable for the gc?

Reply:
There is a flag to disable each collector at  311 & 323

Reply:
I think it'd be cleaner If we can move the instantiation of the `gc` behind an if statement.

```go

if Options.EnableDeregisterInactiveGC || Options.EnableDeletedUnregisteredGC {
....
}
```